### PR TITLE
Uniform Mega-Update

### PR DESCRIPTION
**Pull request by AMcilraith**
**_Things changed:_**
**Replaces the exploratory uniforms with fleet uniforms (to better fit the theme)** 
BoH-Bay\maps\torch\job\outfits

**Removes Exploratory uniforms in the uniform vendor**
**Re-Adds fleet uniforms in the uniform vendor**
BoH-Bay\maps\torch\datums\uniforms_exp_fleet.dm

**Changes some of the definitions for the uniforms**
BoH-Bay\maps\torch\items\clothing

**Changes some of the sprites for the uniforms**
BoH-Bay\maps\torch\icons

### CODE REVIEW

File: maps/torch/datums/uniforms_exp_fleet.dm [hunk 1]
```diff
@@ -25,880 +25,781 @@
...
 
-	utility_under = /obj/item/clothing/under/solgov/utility/expeditionary
-	utility_under_alt = /obj/item/clothing/under/solgov/utility/fleet
+	utility_under = /obj/item/clothing/under/solgov/utility/fleet
 	utility_shoes = /obj/item/clothing/shoes/dutyboots
-	utility_hat = /obj/item/clothing/head/soft/solgov/expedition
-	utility_extra = list(/obj/item/clothing/head/beret/solgov/expedition, /obj/item/clothing/head/ushanka/solgov, /obj/item/clothing/suit/storage/hooded/wintercoat/solgov, /obj/item/clothing/shoes/jackboots/unathi)
-	utility_hat_alt = /obj/item/clothing/head/solgov/utility/fleet
-	utility_extra_alt = list(/obj/item/clothing/head/beret/solgov/fleet, /obj/item/clothing/head/ushanka/solgov/fleet, /obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,/obj/item/clothing/head/soft/solgov/fleet)
+	utility_hat = /obj/item/clothing/head/solgov/utility/fleet
```
Comment (Anton-Kr) [inline maps/torch/datums/uniforms_exp_fleet.dm:39]: Not necessarily against this but note that this does remove some uniform options.

File: maps/torch/datums/uniforms_exp_fleet.dm [hunk 2]
```diff
@@ -25,880 +25,781 @@
...
-	dress_extra_alt = list(/obj/item/clothing/head/beret/solgov/fleet/dress)
+	dress_extra = list(/obj/item/clothing/head/beret/solgov/fleet/dress)
 
 /decl/hierarchy/mil_uniform/fleet/com //Can only be officers
-	name = "NTEF command"
+	name = "SCGF"
 	departments = COM
 
-	utility_under = /obj/item/clothing/under/solgov/utility/expeditionary/officer/command
-	utility_extra = list(/obj/item/clothing/head/beret/solgov/expedition/command,
```
Comment (Anton-Kr) [inline maps/torch/datums/uniforms_exp_fleet.dm:75]: Seems like a fairly big reduction of stuff just because they are "command". Not really happy with it.

File: maps/torch/datums/uniforms_exp_fleet.dm [hunk 3]
```diff
@@ -25,880 +25,781 @@
...
-	service_over_alt = /obj/item/clothing/suit/storage/solgov/service/fleet/officer
-
-	dress_under = /obj/item/clothing/under/solgov/service/expeditionary/command/command
-	dress_skirt = /obj/item/clothing/under/solgov/service/expeditionary/command/skirt/command
-	dress_over = /obj/item/clothing/suit/storage/solgov/dress/expedition/command
-	dress_hat = /obj/item/clothing/head/solgov/service/expedition/command
-	dress_extra = list(/obj/item/clothing/accessory/solgov/ec_scarf, /obj/item/clothing/accessory/cloak/boh/dress/command)
-	dress_over_alt = /obj/item/clothing/suit/storage/solgov/dress/fleet/officer
-	dress_hat_alt = /obj/item/clothing/head/solgov/dress/fleet/command
-	dress_extra_alt = list(/obj/item/weapon/storage/belt/holster/sheath/fleet, /obj/item/clothing/accessory/cloak/boh/command, /obj/item/clothing/head/beret/solgov/fleet/dress/command)
```
Comment (Anton-Kr) [inline maps/torch/datums/uniforms_exp_fleet.dm:104]: All of this should be retained.

File: maps/torch/datums/uniforms_exp_fleet.dm [hunk 4]
```diff
@@ -25,880 +25,781 @@
...
+	service_under = /obj/item/clothing/under/solgov/service/fleet/officer
+	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet/officer
+	dress_hat = /obj/item/clothing/head/solgov/dress/fleet/command
+	dress_extra = /obj/item/weapon/material/sword/officersword
 
 /decl/hierarchy/mil_uniform/fleet/com/seniorofficer
-	name = "NTEF senior command"
-	min_rank = 15
+	name = "SCGF senior command"
+	min_rank = 19
```
Comment (Anton-Kr) [inline maps/torch/datums/uniforms_exp_fleet.dm:80]: What does this change?

File: maps/torch/datums/uniforms_exp_fleet.dm [hunk 5]
```diff
@@ -25,880 +25,781 @@
...
 
 /decl/hierarchy/mil_uniform/fleet/com/capt //Can only be officers
-	name = "NTEF captain"
-	min_rank = 16
-
-	utility_hat = /obj/item/clothing/head/soft/solgov/expedition/co
-	dress_over = /obj/item/clothing/suit/storage/solgov/dress/expedition/command/capt
-	dress_hat = /obj/item/clothing/head/solgov/service/expedition/captain
+	name = "SCGF captain"
+	min_rank = 21
```
Comment (Anton-Kr) [inline maps/torch/datums/uniforms_exp_fleet.dm:88]: Ditto here. What does this change?

File: maps/torch/datums/uniforms_exp_fleet.dm [hunk 6]
```diff
@@ -25,880 +25,781 @@
...
+	dress_hat = /obj/item/clothing/head/solgov/dress/fleet/captain
+	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet/captain
 
 /decl/hierarchy/mil_uniform/fleet/com/flagofficer
-	name = "NTEF flag command"
-	min_rank = 17
+	name = "SCGF flag command"
+	min_rank = 22
 
-	service_over_alt = /obj/item/clothing/suit/storage/solgov/service/fleet/flag
```
Comment (Anton-Kr) [inline maps/torch/datums/uniforms_exp_fleet.dm:128]: I'd prefer they keep the unique flag uniforms, even if not obtainable in game.

File: maps/torch/datums/uniforms_exp_fleet.dm [hunk 7]
```diff
@@ -25,880 +25,781 @@
...
 	dress_over = /obj/item/clothing/suit/storage/solgov/dress/expedition/command/adm
 	dress_hat = /obj/item/clothing/head/solgov/service/expedition/captain
 
 /decl/hierarchy/mil_uniform/fleet/eng
-	name = "NTEF engineering"
+	name = "SCGF engineering"
 	departments = ENG
-
-	utility_under = /obj/item/clothing/under/solgov/utility/expeditionary/engineering
-	utility_extra = list(/obj/item/clothing/head/beret/solgov/expedition/engineering,
```
Comment (Anton-Kr) [inline maps/torch/datums/uniforms_exp_fleet.dm:145]: Retain all of this.

File: maps/torch/datums/uniforms_exp_fleet.dm [hunk 8]
```diff
@@ -25,880 +25,781 @@
...
-						 /obj/item/clothing/suit/storage/hooded/wintercoat/solgov,
-						 /obj/item/clothing/shoes/jackboots/unathi,
-						 /obj/item/clothing/gloves/thick/duty/solgov/eng,)
-	utility_extra_alt = list(/obj/item/clothing/under/solgov/utility/fleet/officer/command,
+	service_under = /obj/item/clothing/under/solgov/service/fleet
+	service_skirt = /obj/item/clothing/under/solgov/service/fleet/skirt
+	service_hat = /obj/item/clothing/head/solgov/dress/fleet
+	service_over = /obj/item/clothing/suit/storage/solgov/service/fleet
+
+	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet
```
Comment (Anton-Kr) [inline maps/torch/datums/uniforms_exp_fleet.dm:160]: Needs to be correct sword+sheath.

File: maps/torch/items/clothing/boh_accessory.dm [hunk 9]
```diff
@@ -28,21 +30,21 @@
...
 /obj/item/clothing/accessory/badge/solgov/tags
 	badge_string = "SGV Dagon"
 
 /obj/item/clothing/accessory/badge/solgov/tags/fleet
-	badge_string = "SGF"
+	badge_string = "SCGF"
 
 /obj/item/clothing/accessory/badge/solgov/tags/marine
-	badge_string = "SMC"
+	badge_string = "SCGMC"
```
Comment (Anton-Kr) [inline maps/torch/items/clothing/boh_accessory.dm:41]: Not sure what we're going with here acronyms wise. I'll consult lore maintainers.

File: maps/torch/datums/uniforms_exp_fleet.dm (hunk 2 above)
Comment (AMcilraith) [reply maps/torch/datums/uniforms_exp_fleet.dm:75]: This prevents there from being two separate uniform types, the expeditionary fleet and the SCGF. This uses the old NTEF uniforms so it was changed to the SGF uniforms

File: maps/torch/items/clothing/boh_clothing.dm [hunk 10]
```diff
@@ -1,60 +1,76 @@
...
-	icon_override = 'maps/torch/icons/mob/onmob_cloak_boh.dmi'
-	accessory_icons = list(slot_tie_str = 'maps/torch/icons/mob/onmob_cloak_boh.dmi', slot_w_uniform_str = 'maps/torch/icons/mob/onmob_cloak_boh.dmi', slot_wear_suit_str = 'maps/torch/icons/mob/onmob_cloak_boh.dmi')
+	icon_state = "cloak_alt"
+	icon_override = 'maps/torch/icons/mob/onmob_cloak.dmi'
+	accessory_icons = list(slot_tie_str = 'maps/torch/icons/mob/onmob_cloak.dmi', slot_w_uniform_str = 'maps/torch/icons/mob/onmob_cloak.dmi', slot_wear_suit_str = 'maps/torch/icons/mob/onmob_cloak.dmi')
 
 /obj/item/clothing/accessory/cloak/boh/dress
 	name = "boatcloak"
-	desc = "A fancy SGF boatcloak with silver ribbon."
+	desc = "A fancy NTEF boatcloak with silver ribbon."
```
Comment (Anton-Kr) [inline maps/torch/items/clothing/boh_clothing.dm:25]: NTEF references.

File: maps/torch/items/clothing/boh_clothing.dm [hunk 11]
```diff
@@ -88,44 +104,93 @@
...
-	/obj/item/weapon/storage/firstaid, /obj/item/weapon/reagent_containers/hypospray, /obj/item/roller, /obj/item/device/suit_cooling_unit, /obj/item/weapon/storage/)
-
-	initial_modules = list(
-		/obj/item/rig_module/cooling_unit,
-		/obj/item/rig_module/maneuvering_jets
-	)
-
-//Psionic Advisor
-
-/decl/hierarchy/outfit/job/torch/crew/command/psiadvisor
```
Comment (Anton-Kr) [inline maps/torch/items/clothing/boh_clothing.dm:113]: Where is this now?

File: maps/torch/items/clothing/boh_hats.dm [hunk 12]
```diff
@@ -0,0 +1,22 @@
+/obj/item/clothing/head/hardhat/self_protection
```
Comment (Anton-Kr) [inline maps/torch/items/clothing/boh_hats.dm:1]: Is this mapped anywhere?

File: maps/torch/items/clothing/boh_masks.dm [hunk 13]
```diff
@@ -0,0 +1,9 @@
+/obj/item/clothing/mask/breath/gill
```
Comment (Anton-Kr) [inline maps/torch/items/clothing/boh_masks.dm:1]: Is this mapped anywhere?

File: maps/torch/items/clothing/solgov-armor.dm [hunk 14]
```diff
@@ -1,12 +1,12 @@
 /obj/item/clothing/suit/armor/vest/solgov
-	name = "\improper SGF armored vest"
-	desc = "A synthetic armor vest. This one is marked with the crest of the SGF."
+	name = "\improper NTF armored vest"
```
Comment (Anton-Kr) [inline maps/torch/items/clothing/solgov-armor.dm:2]: NTF references

File: maps/torch/items/clothing/solgov-hands.dm [hunk 15]
```diff
@@ -3,50 +3,97 @@
 	desc = "You should never see this."
 	item_icons = list(slot_gloves_str = 'maps/torch/icons/mob/onmob_hands_solgov.dmi')
 	icon = 'maps/torch/icons/obj/obj_hands_solgov.dmi'
-	sprite_sheets = list(
```
Comment (Anton-Kr) [inline maps/torch/items/clothing/solgov-hands.dm:6]: This needs to be retained.

File: maps/torch/datums/uniforms_exp_fleet.dm (hunk 3 above)
Comment (AMcilraith) [reply maps/torch/datums/uniforms_exp_fleet.dm:104]: This should not be retained since it is the outdated NTEF uniform, which is replaced by the SGF Uniform already in the code.

File: maps/torch/items/clothing/solgov-hands.dm [hunk 16]
```diff
@@ -3,50 +3,97 @@
...
+/obj/item/clothing/gloves/thick/duty/solgov/fingerless/med
+	name = "fingerless medical duty gloves"
+	desc = "These black duty gloves are made from durable synthetic materials, and have a lovely blue accent color. This pair does not, in fact, protect your fingers."
+	icon_state = "duty_gloves_med_fingerless"
+	item_state = "duty_gloves_med_fingerless"
+
 /obj/item/clothing/gloves/thick/duty/solgov/sec
 	name = "security duty gloves"
-	desc = "These black duty gloves are made from durable synthetic materials, and have a lovely red accent color."
+	desc = "These black duty gloves are made from durable synthetic materials, and have a lovely pale blue accent color."
```
Comment (Anton-Kr) [inline maps/torch/items/clothing/solgov-hands.dm:67]: Vesta C+P is bad.

File: maps/torch/items/clothing/solgov-suit.dm [hunk 17]
```diff
@@ -68,28 +68,45 @@
 
 /obj/item/clothing/suit/storage/solgov/service/fleet
 	name = "fleet service jacket"
-	desc = "A navy blue SolGov Fleet service jacket."
+	desc = "A black Nanotrasen Expeditionary Fleet service jacket."
```
Comment (Anton-Kr) [inline maps/torch/items/clothing/solgov-suit.dm:71]: NTEF reference

File: maps/torch/datums/uniforms_exp_fleet.dm (hunk 4 above)
Comment (AMcilraith) [reply maps/torch/datums/uniforms_exp_fleet.dm:80]: this is a minor wording change, nothing really to it.

File: maps/torch/items/clothing/solgov-suit.dm [hunk 18]
```diff
@@ -442,21 +476,39 @@
 	helmet = /obj/item/clothing/head/helmet/space/void/command
 	boots = /obj/item/clothing/shoes/magboots
 
+//Exploration
+/obj/item/clothing/head/helmet/space/void/exploration
```
Comment (Anton-Kr) [inline maps/torch/items/clothing/solgov-suit.dm:480]: Why is this changed all of the sudden?

File: maps/torch/datums/uniforms_exp_fleet.dm (hunk 6 above)
Comment (AMcilraith) [reply maps/torch/datums/uniforms_exp_fleet.dm:128]: Indeed, this can be fixed
